features.yaml

```diff
@@ -1,299 +1,344 @@
 features:
   - name: "Protocols:HTTP1.1/HTTP2/gRPC/TCP"
+    id: "traffic.httpprotocols"
     level:
       checklist: ""
       maturity: Stable
       nextExpectedPromotion: ""
     area: Traffic Management
   - name: "Protocols:Websockets/MongoDB"
+    id: "traffic.websocketprotocols"
     level:
       checklist: ""
       maturity: Stable
       nextExpectedPromotion: ""
     area: Traffic Management
   - name: "Traffic Control: label/content based routing, traffic shifting"
+    id: "traffic.control"
     level:
       checklist: ""
       maturity: Stable
       nextExpectedPromotion: ""
     area: Traffic Management
   - name: "Resilience features: timeouts, retries, connection pools, outlier detection"
+    id: "traffic.resilience"
     level:
       checklist: ""
       maturity: Stable
       nextExpectedPromotion: ""
     area: Traffic Management
   - name: "Gateway: Ingress, Egress for all protocols"
+    id: "traffic.ingressgateway"
     level:
       checklist: ""
       maturity: Stable
       nextExpectedPromotion: ""
     area: Traffic Management
   - name: "Gateway Injection"
+    id: :"traffic.gatewayinjection"
     level:
       checklist: features/gateway_injection.md
       maturity: Alpha
       nextExpectedPromotion: "1.11"
     area: Traffic Management
   - name: "TLS termination and SNI Support in Gateways"
+    id: "traffic.tls-termination"
     level:
       checklist: ""
       maturity: Stable
       nextExpectedPromotion: ""
     area: Traffic Management
   - name: "SNI (multiple certs) at ingress"
+    id: "traffic.multicertingresssni"
     level:
       checklist: ""
       maturity: Stable
       nextExpectedPromotion: ""
     area: Traffic Management
   - name: "Locality load balancing"
+    id: "traffic.localityloadbalancing"
```

Comment:
We now have two mappings of "features" - the other one being here https://github.com/istio/istio/blob/809c761526f43e05853d1b8f389449a41e7c4d27/pkg/test/framework/features/features.yaml#L36 and doesn't have the same identifiers. it seems like we should have one

Reply:
Yeah, the IDs should probably be the same.

Reply:
Talking to @therealmitchconnors, it sounds like either the other feature IDs will go away, or they will be updated to match the ones here so I think that using the ones here is fine. 

There's also not a one-to-one mapping with what's in the other features.yaml and what's here.